package.json

```diff
@@ -22,16 +21,15 @@
   "devDependencies": {
     "ajv": "^6.0.0",
     "copy-webpack-plugin": "^4.3.1",
-    "css-loader": "^0.28.9",
-    "extract-text-webpack-plugin": "^3.0.2",
+    "css-loader": "^1.0.0",
     "file-loader": "^1.1.6",
-    "node-sass": "4.6.1",
+    "node-sass": "^4.9.3",
     "rebuild-node-sass": "^1.1.0",
     "sass-loader": "^6.0.6",
     "style-loader": "^0.19.1",
-    "uglifyjs-webpack-plugin": "^1.1.6",
     "url-loader": "^0.6.2",
-    "webpack": "^3.10.0"
+    "webpack": "^4.18.0",
+    "webpack-cli": "^3.1.0"
```

Comment:
remove `jquery-ui` entirely; IGV was the only thing depending on it.
remove `extract-text-webpack-plugin`. This was unused, and webpack 4 threw warnings about it being old.
remove `uglifyjs-webpack-plugin`. This is now a default inside of webpack 4.